dev-docs/source/Testing/MuonInterface/MuonTesting.rst

```diff
@@ -1,224 +1,190 @@
+	:alt: Cu-fitting.png
+
+-----------------------------------------------
+
+.. _hifi_transverse_field_simultaneous_fitting:
 
 HIFI Transverse Field Simultaneous Fitting
 ------------------------------------------
 
-*Preparation*
-
--  Files ``HIFI00134028`` – ``HIFI00134039.nxs``
--  Make sure the location of these files is included in your search
-   directories
--  Make sure that the instrument in the Muon Interface is set to
-   ``HIFI``
-
 **Time required 5 - 10 minutes**
 
---------------
-
-1. Open the Muon interface, load the first file ``HIFI00134028``
-
--  Using the ``>`` to the right of the ``Load Data`` box, step through
-   to ``134034``
--  Go to the ``Grouping Options`` tab
--  Click ``Guess Alpha`` - in the ``Pair Table`` box a value of Alpha
-   close to 1.3 should appear
--  Return to ``Home`` tab
--  Click ``>`` until all data sets up to ``134039`` have been loaded
--  In the ``Settings`` tab ensure that ``Enable Multiple Fitting`` is
-   checked
--  Go to the ``Data Analysis`` tab
--  Right-click in the ``Fit Functions`` box
--  Select ``Add function``
--  Choose ``Background``>\ ``Flat Background``
--  Right-click again and ``Add function``
--  Choose ``Muon``>\ ``ExpDecayOsc``
--  In the ``Fit Function`` window select all parameters except
-   ``Frequency`` to be ``Global``
--  In the ``Runs`` box enter ``134028-39``
--  Check the ``Simultaneous`` radio button
--  Select ``Fit``>\ ``Fit``
--  After some time the fir should fail to converge, with a large ``Chi``
-   value (>100) reported in the ``Fit Function`` box.
--  So we want to re-start to see if the fit works a different way
--  In the main Mantid ``Workspaces`` dialog select all the workspaces
-   and delete them
--  Back in the Muon interface load up ``HIFI00134034``
--  Go to ``Grouping Options`` and select ``Guess Alpha``
--  Go to the ``Data Analysis`` tab
--  In the ``Runs`` box enter ``134028-39``
--  Check the ``Simultaneous`` radio button
--  Add the same functions as above (``Flat Background`` and
-   ``ExpDecayOsc``)
--  In the ``Fit Function`` window select all parameters except
-   ``Frequency`` to be ``Global``
--  Click on the value of the ``Frequency`` parameter; a ``...`` box
-   should appear, click it
--  Enter values for each of the rows in the table that opens, values
-   from ``0.0 - 1.1`` in steps of ``0.1``
--  Click ``OK``
--  Select ``Fit``>\ ``Fit``
--  You should now get a significantly lower ``Chi-sq`` value (<10)
--  In the main Mantid window
--  Click on the ``MuonSimulFit_134028-39``
--  Scroll to the workspaces
--  Right-click on one, choose ``Plot spectrum``
--  Choose ``1D Plot`` and select ``Plot All``
--  You should get some plots like this:
-
-.. figure:: ../../images/HIFI-TF-Result.png
-   :alt: HIFI-TF-Result.png
-
+- Open **Muon Analysis** (*Interfaces* > *Muon* > *Muon Analysis*)
+- Change *Instrument* to **HIFI**, found in the *Home* tab
+- Load the first file; ``134028``
```

Comment:
instead of all this stepping we can:
load 134028-39
Go to grouping tab
Click the Guess alpha button
Select 134034 in the pop up
alpha close to 1.3
(then the fitting tab stuff you wrote)